src/include/storage/garbage_collector.h

```diff
@@ -51,19 +56,151 @@ class GarbageCollector {
    */
   uint32_t ProcessUnlinkQueue();
 
```

Comment:
You probably want to order all the functions alphabetically as the original file did. 
I notice that the order of the functions are the same in .h and .cpp file, which is very nice.